docs/introduction.rst

```diff
@@ -118,36 +120,36 @@ More documentation on :doc:`usage/fipreports`.
 ``satfunc``
 ^^^^^^^^^^^
 
-Extracts saturation functions (SWOF, SGOF, etc) from the deck and merges
-into one DataFrame. Can write back to Eclipse include files.
+Extracts saturation functions (SWOF, SGOF, etc) from the :term:`deck` and merges
+into one DataFrame. Can write back to :term:`include files <include file>`.
 
 More documentation on :doc:`usage/satfunc`.
 
 ``equil``
 ^^^^^^^^^
 
 Extracts the information in the `EQUIL` table, `RSVD` and `RVVD` in the
-input deck. Can write back to Eclipse include files.
+:term:`.DATA file`. Can write back to :term:`include files <include file>`.
 
 More documentation on :doc:`usage/equil`.
 
 ``compdat``
 ^^^^^^^^^^^
 
-Extracts well connection data from the `COMPDAT` keyword in the input deck.
+Extracts well connection data from the `COMPDAT` keyword in the :term:`.DATA file`.
 For multi-segment wells, `WELSEGS` and `COMPSEGS` is also parsed. The
 data is available as three different dataframes, which can be merged.
 
-It is also possible to parse individual "include" files, not only a
-finished working deck.
+It is also possible to parse individual :term:`"include files" <include file>`, not only a
+finished working :term:`deck`.
```

Comment:
Use the term complete from the glossary instead. Non need for quotes either since we have defined it in the glossary.

I suggest the following:

"It is also possible to use an incomplete :term:`include file`, but several keywords from the schedule section are read to correctly extract the data."

Reply:
The term complete refers to whether the `deck` contains enough information for the simulator to run as requested. I think maybe the beginning of the sentence came out a bit weird;

```Extracts well connection data from the `COMPDAT` keyword in the :term:`.DATA file`.```

Rewrote the entire thing a bit
```
Extracts well connection data from the `COMPDAT` keyword in the :term:`deck`.
For multi-segment wells, `WELSEGS` and `COMPSEGS` is also parsed. The
data is available as three different dataframes, which can be merged.

It is also possible to parse individual :term:`"include files" <include file>`.
These files do not necessarily have to be part of a complete :term:`deck`
```